examples/Chat/Example06_SimpleChatProtocol.cs

```diff
@@ -23,17 +23,17 @@ public void Example06_SimpleChatProtocol()
                    }
                ]
             }
-            """);
+            """u8.ToArray());
 
         using BinaryContent content = BinaryContent.Create(input);
         ClientResult result = client.CompleteChat(content);
         BinaryData output = result.GetRawResponse().Content;
 
         using JsonDocument outputAsJson = JsonDocument.Parse(output.ToString());
         string message = outputAsJson.RootElement
-            .GetProperty("choices")[0]
-            .GetProperty("message")
-            .GetProperty("content")
+            .GetProperty("choices"u8)[0]
```

Comment:
Aside, but do we know if/when we'll get a `ToDynamicFromJson` extension equivalent?

Reply:
It has been agreed that we would like to have this functionality or something equivalent, but the corresponding work item is currently sitting in the backlog in the face of higher priorities like LROs, pagination, etc. so we don't have an ETA at the moment.